geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -5342,6 +5342,12 @@ public void basicBridgeClientUpdate(DistributedMember serverId, Object key, Obje
     }
   }
 
+  private boolean hasRegisterInterestsWithResultPolicy(boolean isDurable,
+      InterestResultPolicy interestResultPolicy) {
+    return getServerProxy().getPool().getRITracker().hasInterestsWithResultPolicy(fullPath,
```

Comment:
Can serverRegionProxy or pool be null here...I am thinking about the scenario while a thread is doing registration another thread is closing a cache or pool.

Reply:
Well, this is not going to be that case, it is going to be the case of a notification about something on a client/server that is registered. I don't think it can at this point. I have read through the code and I think to have registered interest, you must have those two things be non-null and we would not be calling basicBridgeClientUpdate without a destination aka a pool